apps/relgen/src/index.ts

```diff
@@ -667,12 +683,15 @@ pr.command('review')
       {
         write,
         ruleEval,
+        fileEval,
+        footer,
+        excludedContexts,
       }
     );
 
     output(result, (result) => {
       return dedent`
-        # ${result.summary ?? 'LGTM'}
```

Comment:
The string 'LGTM' is a common abbreviation for 'Looks Good To Me'. While it is not a typo, consider whether this abbreviation is appropriate for the user-facing context. If a more formal or descriptive message is needed, consider expanding it.